### DISCUSSION TITLE
Guild Vs Ambitus?

### DISCUSSION THREAD

**Discussion by StevenDickens** [zopencommunity/meta#29] [General]
This looks like the exact same scope as the Open Mainframe Project Ambitus project? 

https://www.openmainframeproject.org/all-projects/ambitus

Why 2 different attempts to do the same thing?

**Comment by MikeFultonDev** [comment]
@StevenDickens is it active? I clicked on the links from https://github.com/ambitus/ambitus/blob/master/ported_projects.md and got 404's... 
I agree we should work together - we're a small enough community already

**Reply by StevenDickens** [reply]
Not sure on the links, but definitely engage with that team before you stand up a duplicate project with the same mission

**Comment by jbostian** [comment]
Ambitus is active, and Mike and & have had some discussions about the relationship between our two projects.  There are some things that we need to clean up in the Ambitus space, and clarify to avoid confusion about the domain of each community.  I'm in the loop with John Mertic about this as well.

**Comment by MikeFultonDev** [comment]
From my read, I see Ambitus as focused on 'z/OS Open Source' packages, where the code is specific to z/OS. zosopentools is focused on 'open source packages that run many places, and we want to add z/OS to the list'.

**Comment by MikeFultonDev** [comment]
I am thinking we should add some doc around this topic. We are definitely complimentary and we presented on z/OS Open Tools at the last Open Mainframe Summit in Las Vegas.
